### DISCUSSION TITLE
Alert action audit logs/info

### DISCUSSION THREAD

**Discussion by howsTricks** [Security-Onion-Solutions/securityonion#1774] [Ideas] [answered]
Wondering about the Alert tool and auditing actions. Specifically, in our current workflow in a different tool we track who took the action to escalate or ignore. Wondering if there is a way to see this in SO? I checked around the Alert UI as well as Kibana but wasn't able to find something similar.

**Comment by TOoSmOotH** [comment, marked as answer]
We won't have anything like this until RBAC in SOC is implemented. RBAC is on the roadmap but I don't have an ETA at this time.
